### PR TITLE
Remove whole brain from bundle dict for reco viz

### PR DESCRIPTION
**Pull request by 36000**
- [x] move test back to nightly once passing

### CODE REVIEW

**Comment by arokem** [comment]
👍

File: AFQ/tests/test_api.py [hunk 1]
```diff
@@ -281,13 +281,15 @@ def test_AFQ_reco():
     myafq = api.AFQ(
         bids_path=bids_path,
         dmriprep='vistasoft',
+        viz_library="plotly",
```
Comment (36000) [inline AFQ/tests/test_api.py:284]:
```suggestion
        viz_backend="plotly",
```

File: AFQ/viz/utils.py [hunk 2]
```diff
@@ -27,36 +27,48 @@
...
+    "FP": (4, 2), "FA": (0, 2),
+    "CC_ForcepsMinor": (4, 2), "CC_ForcepsMajor": (0, 2),
+    "IFO_L": (4, 1), "IFO_R": (4, 3), "IFOF_L": (4, 1), "IFOF_R": (4, 3),
+    "HCC_L": (4, 0), "HCC_R": (4, 4),
+    "ILF_L": (3, 0), "ILF_R": (3, 4), "F_L": (3, 0), "F_R": (3, 4),
+    "SLF_L": (2, 1), "SLF_R": (2, 3),
+    "ARC_L": (2, 0), "ARC_R": (2, 4), "AF_L": (2, 0), "AF_R": (2, 4),
+    "UNC_L": (0, 1), "UNC_R": (0, 3), "UF_L": (0, 1), "UF_R": (0, 3)})
+
+BUNDLE_RECO_2_AFQ = \
```
Comment (arokem) [inline AFQ/viz/utils.py:65]: This one might be more useful in `AFQ.data`, as we might want it for comparisons that don't have anything to do with visualization.

**Review by arokem — COMMENTED** [review]
Just one comment for now.

File: AFQ/viz/utils.py (hunk 2 above)
Comment (36000) [reply AFQ/viz/utils.py]: Done! In the long run we may want to further break up both files. See #427

**Comment by 36000** [comment]
@arokem  this is ready for review / merge

**Pull request merged**